pkgs/applications/audio/zrythm/default.nix

```
@@ -160,6 +160,7 @@ stdenv.mkDerivation rec {
   preFixup = ''
     gappsWrapperArgs+=(
       --prefix GSETTINGS_SCHEMA_DIR : "$out/share/gsettings-schemas/${pname}-${version}/glib-2.0/schemas/"
+      --prefix XDG_DATA_DIRS : "${breeze-icons}/share/"
```

Alternatively, we have `XDG_ICON_DIRS` (this is Nixpkgs-specific) that can pick up icon themes from `buildInputs`. When a icon theme inherit (propagate) some other icon themes, `XDG_ICON_DIRS` picks up them as well.

```suggestion
      --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS"
```

---

Thank you, this seems to compile a working binary too. 

Force-updated the PR branch.